tokenclient/tokenFlows.go

```diff
@@ -31,6 +33,27 @@ type TokenFlows struct {
 	identity env.Identity
 	Options  Options
 	tokenURI string
+	cache    *cache.Cache
+}
+
+type request struct {
+	http.Request
+	key string `default:""`
```

Comment:
What does the default tag do or where is it used? That's the first time I see this tag.

Reply:
oh yes, I've tested it again, this default value is not required here, if i specify request w/o key - as its by default "".